### PR TITLE
feat(molecule/checkboxField): add alert state

### PR DESCRIPTION
**Pull request by quinwacca**
Add third `state`: `alert`, via `alertText`.

Captura de pantalla 2020-02-03 a las 12 26 06

### CODE REVIEW

File: demo/molecule/checkboxField/demo/index.js [hunk 1]
```diff
@@ -50,6 +50,16 @@ const Demo = () => {
             onChange={(e, {name, value}) => console.log({[name]: value})}
           />
         </li>
+        <li style={styleListItem}>
+          <h2>With Error validation HelpText</h2>
```
Comment (AgonisticKatai) [inline demo/molecule/checkboxField/demo/index.js]: With **ALERT** validation HelpText 😬

File: demo/molecule/checkboxField/playground [hunk 2]
```diff
@@ -85,6 +85,17 @@ return (
           value="In some place of La Mancha which name..."
         />
       </li>
+      <li style={styleListItem}>
+        <h2>
+          With Error validation HelpText
```
Comment (AgonisticKatai) [inline demo/molecule/checkboxField/playground]: With ALERT validation HelpText 😬

**Review by giodelabarrera — APPROVED** [review]

**Review by AgonisticKatai — APPROVED** [review]

**Review by davidmartin84 — APPROVED** [review]

File: demo/molecule/checkboxField/demo/index.js (hunk 1 above)
Comment (quinwacca) [reply demo/molecule/checkboxField/demo/index.js]: Oops! Fixed, thanks! 👍

File: demo/molecule/checkboxField/playground (hunk 2 above)
Comment (quinwacca) [reply demo/molecule/checkboxField/playground]: Oops! Fixed, thanks! 👍

**Review by devictoribero — APPROVED** [review]

**Pull request merged**